library/src/test/java/com/novoda/downloadmanager/BatchBuilderTest.java

```diff
@@ -0,0 +1,41 @@
+package com.novoda.downloadmanager;
+
+import java.util.Collections;
+
+import org.junit.Test;
+
+import static com.google.common.truth.Truth.assertThat;
+
+public class BatchBuilderTest {
```

Comment:
Just wanted to make sure that the `Builder` operated as I expected as I was writing it 😄